GroupProject/Models/PosiblePlayer.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GroupProject.Models
+{
+    public class PosiblePlayer
+    {
+        public int PosiblePlayerId { get; set; }
+
+        [DisplayName("Player")]
+        [Required]
+        [DataType(DataType.Text)]
+        public string PosiblePlayerName { get; set; }
+
+        [DisplayName("User Reviews")]
+        [Required]
+        [DataType(DataType.MultilineText)]
+        public string PlayerComments { get; set; }
+    }
+}
```

Comment:
Created a possible player model so that users can input their information. After doing so I generated a scaffolded item to create the pages they can customize where the users will be able to input their information.
https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model?view=aspnetcore-3.1&tabs=visual-studio